### PR TITLE
Replace math problems with a captcha for bot prevention

### PR DESCRIPTION
**Pull request by Iapetus-11**

### CODE REVIEW

File: bot/cogs/commands/econ.py [hunk 1]
```diff
@@ -72,49 +72,47 @@ def _link_max_concurrency(self):
...
+    async def bot_prevention(self, ctx: Ctx, points: int = 1) -> bool:
+        """
+        If a certain number of bottable commands have been executed, present an image captcha and wait for user
+        to solve captcha before proceeding
+        """
+
+        command_points = await self.karen.bottable_command_execution(ctx.author.id, points)
+
+        if command_points >= 5:
+            captcha_text = "".join(random.choices("ABCDEFGHJKNPQRSTWXYZ", k=4))
```
Comment (jkriste) [inline bot/cogs/commands/econ.py:84]: not sure if intentional but you're missing `I`, `L`, `M`, `O`, `U`, `V`

File: common/data/data.json [hunk 2]
```diff
@@ -2047,5 +2047,11 @@
     "Cyberpunk 2077",
     "Minecraft 1.17",
     "Minecraft 1.18"
+  ],
+  "font_urls": [
+    "https://fonts.google.com/download?family=Mynerve",
+    "https://fonts.google.com/download?family=Playfair%20Display",
+    "https://fonts.google.com/download?family=Gloock",
+    "https://fonts.google.com/download?family=Roboto"
```
Comment (jkriste) [inline common/data/data.json:2055]: i actually really like this font lol

**Review by jkriste — APPROVED** [review]
lgtm otherwise the missing letters

**Comment by Iapetus-11** [comment]
> lgtm otherwise the missing letters

Intentional to avoid confusion :)

File: bot/cogs/commands/econ.py (hunk 1 above)
Comment (Infernum1) [inline bot/cogs/commands/econ.py:84]: imo remove all the vowels to eliminate even the slightest possibility of an inappropriate word occuring in the captcha

**Comment by Infernum1** [comment]
otherwise lgtm

**Comment by Elephoof** [comment]
I second that

**Review by Infernum1 — APPROVED** [review]
otherwise lgtm

File: bot/cogs/commands/econ.py (hunk 1 above)
Comment (Iapetus-11) [reply bot/cogs/commands/econ.py:84]: meh

**Pull request merged**